### PR TITLE
Update terraform docs

### PR DESCRIPTION
**Pull request by jcoglan**
## Changes in this PR

This updates `terraform/README.md` to accurately reflect what we did to deploy to `pentest` most recently, correcting some mistakes and filling in some missing details that might not be obvious to people who've not done this process before.

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -1,62 +1,76 @@
...
+environment.
 
 We store our state in an s3 bucket on the PaaS in the terraform space.
 
 Deploys will happen automatically from Github Actions
 
 ## Manual Deployment
 
-We should avoid manual deployments but the following instructions will allow you to do one.
+We should avoid manual deployments but the following instructions will allow you
```
Comment (mec) [inline terraform/README.md:15]: Could we change this to account for the fact we have to manually deploy the training/pentest environment? Perhaps somehting like:

> We avoid manual deployments on production and staging, but require them on our training environment (pentest).

File: terraform/README.md [hunk 2]
```diff
@@ -1,62 +1,76 @@
...
+- set environment variables for your AWS and CF credentials by creating a file
+  named `deploy-credentials.sh` containing the following:
+
+  ```
+  export AWS_ACCESS_KEY='...'
+  export AWS_SECRET_ACCESS_KEY='...'
+  export CF_USER='...'
+  export CF_PASSWORD='...'
+  ```
+
```
Comment (mec) [inline terraform/README.md:42]: Could we add somehitng here like 'Never commit sensitive information like keys, secrets or passwords'?

**Review by mec — CHANGES_REQUESTED** [review]
This is so much better. Thank you for taking the time to do this. I left two comments I think we should address.

**Comment by jcoglan** [comment]
@mec I've made the changes you requested now.

**Review by mec — APPROVED** [review]
🚢

**Pull request merged**